### PR TITLE
[FRE-1314] fix: filter available wallets in RenderWalletList component

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

**Comment by codingki** [comment]
also found some unnecessary hook deps

**Comment by ericHgorski** [comment]
> Not working for solana wallets image

This is as intended if only one wallet option is available then we should all available options. I could refine the code so this is only the case with walletconnect and not any length === 1 available wallet list. Its a design question if we're ok with this modal having just a single option.

**Comment by codingki** [comment]
Ok nice, lgtm if the hook deps fixed

File: packages/widget/src/components/RenderWalletList.tsx [hunk 1]
```diff
@@ -106,78 +114,62 @@ export const RenderWalletList = ({
...
         />
       );
     },
     [connectMutation]
   );
 
   const height = useMemo(() => {
-    return Math.min(530, walletList.length * (ITEM_HEIGHT + ITEM_GAP));
+    return Math.min(530, displayWallets.length * (ITEM_HEIGHT + ITEM_GAP));
   }, [walletList]);
```
Comment (codingki) [inline packages/widget/src/components/RenderWalletList.tsx:170]:
```suggestion
  }, [displayWallets]);
```

**Review by codingki — APPROVED** [review]
merge when eslint deps fixed

**Pull request merged**